Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -166,14 +167,15 @@ public static async Task<(int totalCount, int newCount)> ImportAccountAsync(Func
                 stop();
                 var putBreakPointHere = logOutput;
                 ScanEnd?.Invoke(null, null);
+                GC.Collect(GC.MaxGeneration, GCCollectionMode.Aggressive, true, true);
             }
         }
 
         private static async Task<List<ImportItem>> scanAccountsAsync(Func<Account, Task<ApiExtended>> apiExtendedfunc, Account[] accounts, LibraryOptions libraryOptions)
         {
             var tasks = new List<Task<List<ImportItem>>>();
 
-            using LogArchiver archiver
+            await using LogArchiver archiver
```

Comment:
I have never seen not contemplated `await using`; that's wild. What's being awaited? Isn't this just a ctor?

Reply:
It's actually a waiting [DisposeAsync](https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-disposeasync).

ZipArchive writes all changes to disk when it's disposed, so it's good to make it async.